aodntools/timeseries_products/hourly_timeseries.py

```diff
@@ -595,11 +595,9 @@ def hourly_aggregator(files_to_aggregate, site_code, qcflags, file_path ='./'):
     ncout_filename = generate_netcdf_output_filename(nc=nc_aggregated, facility_code=facility_code, data_code=data_code,
                                                      site_code=site_code,
                                                      product_type=product_type, file_version=file_version)
-    ncout_path = os.path.join(file_path, ncout_filename)
-
-    write_netCDF_aggfile(nc_aggregated, ncout_path, encoding, file_path)
```

Comment:
To me it makes more sense the way you had it before. Just create `ncout_path` _once_ by doing the join, then that's all you need to supply to `write_netCDF_aggfile`. That function doesn't separately need to know what the `file_path` is.

Reply:
:+1: